index.html

```diff
@@ -226,6 +229,18 @@ <h1 class="title">About Myself</h1>
             </div>
         </div>
     </section>
+
+    <section class="contact-form">
+        <h1 class="form-header">Contact me</h1>
```

Comment:
**OPTIONAL** - To help improve the SEO of your site, I advise that you have only one `h1` tag on your HTML page. Please fix this 🙏

Reply:
So Only one h1 tag on My HTML page, meaning all the other h1 in the about section and form sections i used should be replaced?

Reply:
Yes, it means exactly this :+1: